webcompat/static/js/lib/issues.js

```diff
@@ -325,15 +325,16 @@ issues.StateButtonView = Backbone.View.extend({
     } else {
       this.model.toggleState();
     }
-  }
+  } 
 });
 
 issues.MainView = Backbone.View.extend({
   el: $(".js-Issue"),
   events: {
     "click .js-Issue-comment-button": "addNewComment",
     "click": "closeLabelEditor",
-    "click .wc-Comment-content-nsfw": "toggleNSFW"
+    "click .wc-Comment-content-nsfw": "toggleNSFW",
+    "keydown": "keyAction"
```

Comment:
Can we give this a more descriptive name, like `handleKeyShortcuts` (or something better)?